internal/pkg/cli/job_init.go

```diff
@@ -121,6 +175,110 @@ func (o *initJobOpts) Execute() error {
 	return nil
 }
 
+func (o *initJobOpts) askJobType() error {
+	o.JobType = manifest.ScheduledJobType
+	return nil
+}
+
+func (o *initJobOpts) askJobName() error {
+	if o.Name != "" {
+		return nil
+	}
+
+	name, err := o.prompt.Get(
+		fmt.Sprintf(fmtWkldInitNamePrompt, color.Emphasize("name"), color.HighlightUserInput(o.JobType)),
+		fmt.Sprintf(fmtWkldInitNameHelpPrompt, job, o.AppName()),
+		validateSvcName,
+		prompt.WithFinalMessage("Job name:"),
+	)
+	if err != nil {
+		return fmt.Errorf("get job name: %w", err)
+	}
+	o.Name = name
+	return nil
+}
+
+func (o *initJobOpts) askDockerfile() error {
+	if o.DockerfilePath != "" {
+		return nil
+	}
+	df, err := askDockerfile(o.Name, o.fs, o.prompt)
+	if err != nil {
+		return err
+	}
+	o.DockerfilePath = df
+	return nil
+}
+
+func (o *initJobOpts) askSchedule() error {
```

Comment:
Also for this method it is to prompt users to get the value for `o.Schedule`. What do you think of moving the logic of `askSchedule` to `selector` pkg as well and then `askSchedule` will be crystal clear like this:
``` go
func (o *initJobOpts) askSchedule() error {
	if o.Schedule != "" {
		return nil
	}
	schedule, err := o.JobSel.Schedule(
		jobInitSchedulePrompt,
		jobInitScheduleHelp,
		prompt.WithFinalMessage("Schedule type:"),
	)
	if err != nil {
		return fmt.Errorf("get schedule: %w", err)
	}
	o.Schedule = schedule
        return nil
}
```
What do you think?